<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshComp</class>
 <widget class="ComponentGroup&lt;Raz::Mesh&gt;" name="MeshComp">
  <property name="windowTitle">
   <string>Mesh</string>
  </property>
  <property name="title">
   <string>Mesh</string>
  </property>
  <layout class="QFormLayout" name="meshLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="vertexCountLabel">
     <property name="text">
      <string>Vertex count</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="vertexCount">
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="triangleCountLabel">
     <property name="text">
      <string>Triangle count</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="triangleCount">
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="meshFileLabel">
     <property name="text">
      <string>Mesh file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="FileWidget&lt;FileType::MESH&gt;" name="meshFile">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComponentGroup&lt;Raz::Mesh&gt;</class>
   <extends>QGroupBox</extends>
   <header>RaZor/Interface/ComponentGroup.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileWidget&lt;FileType::MESH&gt;</class>
   <extends>QLineEdit</extends>
   <header>RaZor/Interface/FileWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
